spec/features/authentication/sign_up_spec.rb

```diff
@@ -26,7 +26,7 @@
     within("form") do
       # Errors show below the inputs, Capybara will show the error as follows
       expect(page).to have_content("Emailcan't be blank")
-      expect(page).to have_content("Passwordcan't be blank")
+      expect(page).to have_content("PasswordMinimum is #{Rails.configuration.devise.password_length.min} characterscan't be blank")
```

Comment:
Do we need to check for the specific text though? Could you not check for that input and the fact that there is an error shown?